test_models.py

```
@@ -0,0 +1,361 @@
+        
+        return device
+
+
+async def test_audit_model(device: Device):
+    """Test DataCollectionAudit model creation and relationship."""
+    print("🧪 Testing DataCollectionAudit model...")
+    
+    async with get_async_session() as session:
+        # Create audit record
+        audit = DataCollectionAudit(
+            time=datetime.now(timezone.utc),
```

Ruff (0.12.2)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)